debian/jitsi-meet-web-config.postinst

```diff
@@ -212,6 +216,33 @@ case "$1" in
             fi
 
             invoke-rc.d nginx reload || true
+        elif [[ "$FORCE_OPENRESTY" = "true" && ( -z "$JVB_HOSTNAME_OLD" || "$RECONFIGURING" = "true" ) ]] ; then
+
+            # this is a reconfigure, lets just delete old links
+            if [ "$RECONFIGURING" = "true" ] ; then
+                rm -f /etc/jitsi/meet/$JVB_HOSTNAME_OLD-config.js
+            fi
+
+            # create conf
+            if [ ! -f /etc/jitsi/openresty/$JVB_HOSTNAME.conf ] ; then
+                mkdir -p /etc/jitsi/openresty
+                cp /usr/share/jitsi-meet-web-config/jitsi-meet.example /etc/jitsi/openresty/$JVB_HOSTNAME.conf
+                sed -i "s/jitsi-meet.example.com/$JVB_HOSTNAME/g" /etc/jitsi/openresty/$JVB_HOSTNAME.conf
+            fi
+
+            if [ "$CERT_CHOICE" = "$UPLOADED_CERT_CHOICE" ] ; then
+                # replace self-signed certificate paths with user provided ones
+                CERT_KEY_ESC=$(echo $CERT_KEY | sed 's/\./\\\./g')
+                CERT_KEY_ESC=$(echo $CERT_KEY_ESC | sed 's/\//\\\//g')
+                sed -i "s/ssl_certificate_key\ \/etc\/jitsi\/meet\/.*key/ssl_certificate_key\ $CERT_KEY_ESC/g" \
+                    /etc/jitsi/openresty/$JVB_HOSTNAME.conf
+                CERT_CRT_ESC=$(echo $CERT_CRT | sed 's/\./\\\./g')
+                CERT_CRT_ESC=$(echo $CERT_CRT_ESC | sed 's/\//\\\//g')
+                sed -i "s/ssl_certificate\ \/etc\/jitsi\/meet\/.*crt/ssl_certificate\ $CERT_CRT_ESC/g" \
+                    /etc/jitsi/openresty/$JVB_HOSTNAME.conf
+            fi
+
+            invoke-rc.d openresty reload || true
```

Comment:
As this reuses the nginx template and only the reload command is different. You better merge the logic with the nginx one so we do not duplicate those sed commands.

Reply:
not exactly.
default openresty installation:
1) doesn't provide `/etc/nginx` at all (it resides in `/usr/local/openresty/nginx/conf` (with `/etc/openresty` symlink to it in last versions).
2) even there, there is no `sites-{available,enabled}` thing. There is just default `nginx.conf` (real ustream nginx default, without includes). So, it is supposed that users will manage configuration themselves, using their favourite configuration/includes scheme or config generators. So, it is supposed they will use jitsi config either as example or just include it from it's path.

That's why I didn't reuse NginX's block.

Reply:
So this will produce a broken deployment.

In a similar scenario we add include in prosody config. I think we should do the same here. 
You can put the config then in `/etc/openresty` and include it.

Reply:
https://github.com/jitsi/jitsi-meet/blob/ed139f53ca883573d252b166d052ba80b25790e9/debian/jitsi-meet-prosody.postinst#L134

Reply:
Ok, I'll make it that way (although, monkeypatching nginx config is a hardmode quest :-D)

Reply:
Yep, but the goal is to have a running deployment at the end. Also during installation we need to make sure that Let's encrypt install continues to work which uses the webserver.

Reply:
Yeah, I get it.

Although, on other hand, I personally sometimes use non-standard configurations, and so don't really like when any software trying to "do the best" and patching it, and fails because of non-standard things.

Ideally, I'd like that jitsi had some button to say it "don't touch my configs (not only webserver, but also prosody and coturn) at all, please, I'll set all that up myself, lookin into examples".
But I'm not ready yet to invent that, so I'll just make it as we talked above.